hips/hip-0001.md

```diff
@@ -0,0 +1,427 @@
+---
+hip: 1
+title: "Writing a HIP"
+authors: [ "Matthew Fisher <matt.fisher@microsoft.com>" ]
+created: "2020-08-13"
+type: "process"
+status: "draft"
+---
```

Comment:
The preamble here does not have the `yaml` qualifier.  It because a table in markdown.  Was that on purpose?  The table looks good actually, but I'm not sure it would with all the fields allowed in the preamble.
See this page to see the output of this file: 
https://github.com/bacongobbler/helm-community/blob/writing-a-proposal/hips/hip-0001.md

Reply:
The `yaml` qualifier is not necessary here. The style is equivalent to Hugo's front matter format: https://gohugo.io/content-management/front-matter/

Github notices this as yaml front-matter, and renders it as a table.

See also https://github.com/helm/helm-www/blob/master/content/en/docs/faq/_index.md

Reply:
Ah, makes sense.
I don't have a suggestion to do things differently.  The table looks good.  We can see as we write proposals if this needs any changes.